app/src/main/java/com/amaze/filemanager/database/UtilsHandler.java

```diff
@@ -86,18 +102,18 @@ public void onCreate(SQLiteDatabase db) {
         db.execSQL(queryGrid);
         db.execSQL(queryBookmarks);
         db.execSQL(querySmb);
+//        db.execSQL(querySftp);
```

Comment:
I think you need this line to create the database for new users...

Reply:
Oops...